app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @all_tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+  
+  get '/tweets/new' do
+    if logged_in?
+      current_user
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+  
+  post '/tweets' do
+    if params[:content] != "" 
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.